app/scripts/lib/util.ts

```diff
@@ -142,6 +142,19 @@ function checkAlarmExists(alarmList: { name: string }[], alarmName: string) {
   return alarmList.some((alarm) => alarm.name === alarmName);
 }
 
+/**
+ * Check if string passed is valid hexadecimal
+ *
+ * @param str
+ * @returns boolean
+ */
+function isHexadecimal(str: string): boolean {
```

Comment:
ethereumjs-util has an `isHexString()` method...we are already using it in `shared/modules/hexstring-utils.ts`

Reply:
Good point @segun, I did code changes to include ones from `ethereumjs-util`.